ci-operator/step-registry/origin/e2e/vsphere/origin-e2e-vsphere-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: origin-e2e-vsphere
+  steps:
+    pre:
+    - chain: ipi-conf-vsphere
+    - chain: ipi-install-vsphere
+    test:
+    - ref: origin-e2e-test
+    post:
+    - chain: ipi-deprovision
```

Comment:
this wouldn't have the trust for ci vcenter right? so the destroy calls are probably going to fail?

Reply:
> this wouldn't have the trust for ci vcenter right? so the destroy calls are probably going to fail?

yes! good catch. the image needs to be changed.